lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -31,12 +34,20 @@ export default function AssignmentActivity() {
     assignmentId: string;
     organizationPublicId?: string;
   }>();
-  const assignment = useAPIFetch<AssignmentWithMetrics>(
+
+  const { filters, updateFilters } = useDashboardFilters();
+  const { studentIds } = filters;
+  const search = useSearch();
+  const [, navigate] = useLocation();
+
+  const assignment = useAPIFetch<AssignmentWithCourse>(
```

Comment:
The type here was wrong. This endpoint does not return assignment metrics, but it includes the course information, so I defined a new narrower `AssignmentWithCourse` type.